src/components/create-appointment/helpers.ts

```diff
@@ -0,0 +1,10 @@
+export const calculateEndTime = (
```

Comment:
add example of input and output params, in other case its too long to understand why it needed when someone will work with it. are you sure that this helper needed only in create appointment and make it global useless?

Reply:
will fix and make it global